test/action.ts

```diff
@@ -179,12 +179,7 @@ describe("correctly select GitHub release based on version constraints", async (
 
   it("check latest prerelease", async () => {
     let tag = await getTag(token, "engineerd/setup-kind", "latest", true);
-    assert.equal(tag, "v0.4.0");
-  });
-
-  it("check latest non-prerelease", async () => {
```

Comment:
That specific repo does not have any more non-pre-releases.
We should update the test with another repository that does.